LabManagement/PowerShell/JITUserQuota/JITQuotaWithScheduledTask.ps1

```diff
@@ -0,0 +1,100 @@
+# Lets stop the script for any errors
+$ErrorActionPreference = "Stop"
+
+# ************************************************
+# ************ FIELDS TO UPDATE ******************
+# ************************************************
+
+# List of lab names that we should not include when updating quota)
+$excludeLabs = @('*test*','*demo*','*training*', '*how to*')
+
+# Number of available hours we reset the student to when running this script
+$usageQuota = 8
+
+# Segment of labs to update based on lab plans, regular expression to match
+
+# Match for all lab plans   
+$labPlanNameRegex = "^.*"
+
+# Subscription ID
+$subId = "1111-1111-1111-11111-11111111"
+
+# ************************************************
+
+if ($PSVersionTable.PSEdition -eq 'Desktop' -and (Get-Module -Name AzureRM -ListAvailable)) {
```

Comment:
I think instead of installing for the user-  we should just add a requires at the top for the Az and Az.LabServices modules...

Reply:
I left it this way based on the existing script we had provided to the customer that did this for them.